pinot-core/src/main/java/org/apache/pinot/core/data/table/IndexedTable.java

```diff
@@ -33,43 +32,44 @@
  */
 public abstract class IndexedTable implements Table {
 
-  List<AggregationFunction> _aggregationFunctions;
+  AggregationFunction[] _aggregationFunctions;
   int _numAggregations;
   DataSchema _dataSchema;
 
+  int _capacity;
```

Comment:
A comment on what the difference is between 'capacity' and 'maxCapacity' will be nice. If I get it right, capacity is the number we need to trim to (at most) and maxCapacity is how high we are going to go in order to not drop new keys coming in.

Reply:
That's right. Added the comment in the IndexedTable